style.css

```diff
@@ -382,184 +379,252 @@ input :placeholder {
   font-size: 15px;
   line-height: 24px;
 }
+
 textarea {
   font-family: "Poppins", sans-serif;
   padding: 15px 10px;
   font-size: 17px;
   color: black;
 }
+
 textarea:focus {
   outline: 0;
 }
 
-@media (min-width: 768px) { 
-  html, body {
+@media (min-width: 768px) {
+  html,
+  body {
     margin: 0;
     padding: 0;
   }
-  .FirstSection{
+
+  .DesktopNavtext a:visited {
+    color: var(--logocolor);
+  }
```

Comment:
- [ ] According to the requirement you must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly use the same color as on the original design `color: #344563;`